test/realizations/catchments/Partition_Parser_Test.cpp

```diff
@@ -0,0 +1,67 @@
+#include "gtest/gtest.h"
+#include <stdio.h>
+#include "Partitions_Parser.hpp"
+
+using namespace realization;
+
+class PartitionsParserTest: public ::testing::Test {
+
+    protected:
+
+    PartitionsParserTest() {
+
+    }
+
+    ~PartitionsParserTest() override {
+
+    }
+
+    void SetUp() override;
+
+    void TearDown() override;
+
+    void setupArbitraryExampleCase();
+
+};
+
+void PartitionsParserTest::SetUp() {
+    setupArbitraryExampleCase();
+}
+
+void PartitionsParserTest::TearDown() {
+
+}
+
+void PartitionsParserTest::setupArbitraryExampleCase() {
+
+}
+
+TEST_F(PartitionsParserTest, TestFileReader)
+{
+  const std::string file_path = "/home/shengting.cui/MPI/ngen/data/example_partition_config.json";
```

Comment:
Using an absolute path like this will cause problems in other environments.

Reply:
Will change that.